packages/react-storage/src/components/StorageBrowser/controls/hooks/useActionStart.tsx

```diff
@@ -0,0 +1,20 @@
+import { ActionStartProps } from '../../composables/ActionStart';
+import { useControlsContext } from '../../controls/context';
+
+export type UseActionStart = () => {
+  props?: ActionStartProps;
```

Comment:
I think we're planning to remove additional nesting `props` since it's not required atm. See review comments [here](https://github.com/aws-amplify/amplify-ui/pull/5939/files#r1811676247) and [here](https://github.com/aws-amplify/amplify-ui/pull/5939/files#r1811683437)